cygwin_conv_to_posix_path is deprecated in 1.7 and absent on x86_64.

--- origsrc/Proc-ProcessTable-0.48/os/cygwin.c	2013-02-18 17:34:31.000000000 -0600
+++ src/Proc-ProcessTable-0.48/os/cygwin.c	2013-08-02 04:06:17.625244800 -0500
@@ -218,7 +218,7 @@ OS_get_table()
 	{
 	  char *s;
 	  pname[0] = '\0';
-	  cygwin_conv_to_posix_path (p->progname, pname);
+	  cygwin_conv_path (CCP_WIN_A_TO_POSIX, p->progname, pname, MAX_PATH);
 	  s = strchr (pname, '\0') - 4;
 	  if (s > pname && strcasecmp (s, ".exe") == 0)
 	    *s = '\0';
